src/components/Todos.js

```diff
@@ -2,22 +2,30 @@ import React, { Component } from 'react';
 import Todo from './Todo';
 
 class Todos extends Component {
-  render() {
-    // JSX supports passing in element arrays
-    const array = [
-      <Todo label="First todo..." />,
-      <Todo label="Second todo..." />,
-      <Todo label="Third todo..." />
-    ];
-
-    return (
-      <section className="main">
-        <ul className="todo-list">
-          {array}
-        </ul>
-      </section>
-    );
-  }
+	constructor(props) {
+		super(props);
+	}
+	
+	render() {
+		// JSX supports passing in element arrays
+		let array = Array.from(this.props.items, item => 
+			<Todo 
+				key={item.label}
```

Comment:
I would recommend using a unique ID for the key.  However, i do not advise using the index of the element as that can change.  I would use an ID generator like this...
```
let id = 1;
const getId = () => id++;
```